dvc/command/run.py

```diff
@@ -225,6 +226,12 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help=argparse.SUPPRESS,
     )
+    run_parser.add_argument(
+        "--external",
+        action="store_true",
+        default=False,
+        help="Allow outputs that are outside of the DVC project.",
```

Comment:
Do we use `project`? Maybe:
```suggestion
        help="Allow outputs that are outside of the DVC repo.",
```

Or, simply `Allow external outputs`?